src/Collections-Sequenceable/OrderedCollection.class.st

```diff
@@ -408,6 +415,21 @@ OrderedCollection >> copyEmpty [
 	^self species new
 ]
 
+{ #category : #copying }
+OrderedCollection >> copyForTransform [
```

Comment:
The name copy is misleading. It does not return a copy really. Maybe it can be another speciesForTransform (returning a class like a species)?

But maybe it's ok. I dont see the another name

Reply:
Ooooh, I _like_ that idea. You're right, it's not really a copy—I boxed myself in to that way of thinking because it's being used in place of `copyEmpty[:]`. Having a single `speciesForTransform` would eliminate the duplication between known- and unknown-size (`copyForTransform:` with a size argument and unary `copyForTransform` respectively) as well. Updated!